## üëã Welcome!
**I am Isabel Ruiz.**
**Bioinformatician and Evolutionary Biologist.**

## About Me
I am a researcher with a great curiosity in the evolutionary history of non-model organisms, with a particular interest in the evolution of sensing and perception in the context of multicellularity. Eukarya enthusiastic! I hold a MSc. Bioinformatics form the University of Bristol (2023/24), graduated with distinction, and a BSc. Biology from the Cayetano Heredia Peruvian University. I am eager to help advance and refine workflows for next-generation sequencing (NGS) analysis and their applications in genomics and evolutionary biology.

Relevant experience:
* Designed and implemented a bioinformatics workflow to extract genes of interest from Illumina sequencing data in a Unix environment, to  streamline gene extraction and enhanced future genomic in molluscs (see more in this [GitHub Repository](https://github.com/sciruiz/euk-illumina-gene-extraction-pipeline)) and automatization of phylogenetic analyses (see more in [Kawsay Tree](https://github.com/sciruiz/KawsayTree)) at San Marcos University with *Dr. Rer. Nat.* Pedro E. Romero. 
* Supported ecological research on the Andean bear by contributing to organising and pre-processing of the data that informed habitat modeling and conservation strategies in northern Peru with the [Institute for Research in Ecology and Conservation](https://www.iiecco.org).
* Co-designed and led practical sessions on phylogenetic reconstruction, data mining, and R-based analysis, improving students‚Äô practical understanding of evolutionary methods and computational tools (see more in [EvoTutorials](https://sciruiz.github.io/evotutorials)), at the Cayetano Heredia Peruvian University with *Dr. Rer. Nat.* Armando Valdes.

You can learn more about my professional experience on [LinkedIn](https://www.linkedin.com/in/sciruiz/).
If you would like to contact me, please complete this [Google Form](https://forms.gle/ctKMwxgKzokTyNzD6). I'll be in touch soon!

## Programming Languages
![Python](https://img.shields.io/badge/Python-3776AB?logo=python&logoColor=white)
![Linux](https://img.shields.io/badge/Linux-FCC624?logo=linux&logoColor=black)
![R](https://img.shields.io/badge/R-276DC3?logo=r&logoColor=white)
![Bash](https://img.shields.io/badge/Bash-4EAA25?style=flat-square&logo=gnubash&logoColor=white)
![HTML](https://img.shields.io/badge/HTML5-E34F26?style=flat-square&logo=html5&logoColor=white)

## Skills 
![NGS](https://img.shields.io/badge/NGS-blue?style=flat-square&logo=dna&logoColor=white)
![Phylogenomics](https://img.shields.io/badge/Phylogenetics-green?style=flat-square&logo=tree&logoColor=white)
![Genomics](https://img.shields.io/badge/Genomics-purple?style=flat-square&logo=biolink&logoColor=white)
![Transcriptomics](https://img.shields.io/badge/Transcriptomics-orange?style=flat-square&logo=files&logoColor=white)

## Research interests
* Evolution of sensing and perception, particularly the role of gene duplications in the emergence of novel traits.
* Integration of multi-omics data (genomics, transcriptomics, and single-cell sequencing) to reconstruct evolutionary changes in a stepwise.
* Large-scale phylogenetic analyses to uncover the natural history of life, combining with artificial intelligence to refine analytical methodologies.

_‚ÄúAn understanding of the natural world is a source of not only great curiosity, but great fullfilment.‚Äù ‚Äì David Attenborough_



